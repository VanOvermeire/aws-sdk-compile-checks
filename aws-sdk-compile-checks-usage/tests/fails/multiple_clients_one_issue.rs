use aws_sdk_compile_checks_macro::required_props;

#[required_props]
async fn dynamo_and_sqs(sqs_client: aws_sdk_sqs::Client, dynamodb_client: aws_sdk_dynamodb::Client) {
    let _ = sqs_client.receive_message()
        .queue_url("")
        .send()
        .await;
    let _ = dynamodb_client
        .create_global_table()
        .global_table_name("...")
        .send()
        .await;
}

fn main() {}
