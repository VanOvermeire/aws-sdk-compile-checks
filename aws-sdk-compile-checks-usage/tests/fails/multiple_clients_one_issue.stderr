error: method `create_global_table` is missing required argument(s): `replication_group`
  --> tests/fails/multiple_clients_one_issue.rs:10:10
   |
10 |         .create_global_table()
   |          ^^^^^^^^^^^^^^^^^^^
